src/software/ai/motion_constraint/motion_constraint.h

```diff
@@ -15,7 +15,7 @@ MAKE_ENUM(MotionConstraint,
           FRIENDLY_DEFENSE_AREA,
           // The box around the enemy goal
           ENEMY_DEFENSE_AREA,
-          // The box around the enemy goal, inflated by 0.2 meters for certain situations
+          // The box around the enemy goal, inflated by 0.3 meters for certain situations
```

Comment:
Where is `0.3` coming from here? It seems likely that this comment will quickly become out-of-date again.

Reply:
yeah thoughts on making this a dynamic param?

Reply:
Maybe we shouldn't comment the values of expansion / modification in the enum definition? The inflation amount is more of an implementation detail.

Reply:
removed the 0.2 meters part